api/app/signals/apps/signals/migrations/0026_history.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.1.2 on 2018-11-09 13:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('signals', '0025_create_history_view_in_db'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='History',
+            fields=[
+                ('identifier', models.CharField(max_length=255, primary_key=True, serialize=False)),
```

Comment:
Waarom heb je een primary key nodig? Het is een view

Reply:
Ik heb die primary keys toegevoegd omdat Django ze wilde hebben.

Reply:
Zie comment bij vorige conversation